lib/pages/farm/components/farm_item.dart

```diff
@@ -38,46 +38,50 @@ Widget farmItem(
       1,
       Colors.grey[600]!,
     ),
-    child: Column(
-      mainAxisAlignment: MainAxisAlignment.spaceBetween,
-      children: [
-        farmTitleWidget,
-        // Current Balance
-        Row(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          children: [
-            Text(
-              'Available Balance',
-              style: txStyle,
-            ),
-            Obx(
-              () => Text(
-                '${farm.stakingInfo.value.viewAmount} ${farm.strStakedSymbol}',
+    child: FittedBox(
```

Comment:
Not against using a `FittedBox()` but this widget is scaling down all the contents including the title which makes the metrics centered and we do not want that. Take a look at the `farmTitleWidget` and try to use the `FittedBox()` over there. I would suggest wrapping the text widget with a `SizedBox()` and then wrapping that with a `FittedBox()`. We want to do this because this will make the contents of the farm title be constrained to the `SizedBox()` that we will have defined there. Therefore, the contents will scale within the constraints given by the parent widget.